spec/rails_helper.rb

```diff
@@ -17,6 +18,17 @@ module Features
   config.infer_base_class_for_anonymous_controllers = false
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = false
+
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.around(:each) do |example|
+    DatabaseCleaner.cleaning do
+      example.run
+    end
+  end
```

Comment:
DatabaseCleaner configurations should live in `spec/support/database_cleaner.rb`